.gitignore

```diff
@@ -21,3 +21,6 @@ wiki/.vuepress/dist
 
 # Temp files for dev build
 .temp
+
+# Visual Studio-created directory for... something?
+/.vs
```

Comment:
This is from opening in Visual Studio?
I typically use VS Code for webdev related projects

Reply:
Yes, the `.vs` directory is created from opening it in Visual Studio. I'm not entirely sure what is stored in there, but I added it to `.gitignore` since it was not necessary for the wiki.

I used Visual Studio just because I essentially have it open 24/7.